draft-bbs-signatures.md

```diff
@@ -245,6 +245,27 @@ Procedure:
 8. return SK
 ```
 
+## MapToScalar
+
+MapToScalar algorithm takes an arbitrary value and maps it to a scalar value. 
+
+Data can be mapped multiple ways depending on how it will be used with other Zero-Knowledge Proofs. For example, it is not useful to hash an integer to a scalar using SHA256 and expect it to work properly with Range Proofs.
```

Comment:
```suggestion
Data to be signed can be mapped multiple ways depending on how it will be used downstream in various Zero-Knowledge Proof protocols. For example to be able to construct a range proof for a signed message, it must have been encoded in a particular way during the signing procedure.
```